Packs/ApiModules/Scripts/JSONFeedApiModule/README.md

```diff
@@ -0,0 +1,12 @@
+To use the common JSON feed API logic, run the following command to import the `JsonFeedApiModule`.
+
+```python
+def main():
+    ...
+
+
+from JsonFeedApiModule import *  # noqa: E402
+
+if __name__ in ["builtins", "__main__"]:
+    feed_main()
```

Comment:
shouldn't this call the main and then the main call the feed_main or some other logic?